README.rst

```diff
@@ -1,29 +1,63 @@
 AppImage docs
 =============
 
-|ci_status| |cd_status|
-
-.. |cd_status| image:: https://drone.assassinate-you.net/api/badges/AppImage/docs.appimage.org/status.svg
-   :alt: CD status
-   :target: https://drone.assassinate-you.net/AppImage/docs.appimage.org
-
-.. |ci_status| image:: https://github.com/AppImage/docs.appimage.org/workflows/CI/badge.svg
+.. image:: https://github.com/AppImage/docs.appimage.org/workflows/CI/badge.svg
```

Comment:
Why remove the alt text?

Reply:
What alt text did I remove? As far as I can tell, I only removed the CD status badge as it doesn't exist anymore.